### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by cuauh-cabrera**
Below are listed the features included in this branch:

- [ ] Add index.html
- [ ] Add styles.css
- [ ] Add project description

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                        procesos de conservación integrando a los ecosistemas y a las comunidades a un nivel de paisaje
+                        en el cual interactuan de manera sinérgica ecosistemas, personas, comunidades, cultivos,
+                        conocimientos, culturas y tradiciones.</p>
+                </div>
+                <div class="contenedor-index--pic">
+                    <img src="./assets/lago_optimized.jpeg" alt="lago de valsequillo en puebla, mexico">
+                </div>
+            </section>
+            <section class="hero-2"></section>
+        </article>
```
Comment (cuauh-cabrera) [inline index.html:52]: [CHANGE REQUIRED] Well done using a <section> tag for your hero section, however, you don’t need to enclose your <section> inside an <article>.  Use the <article> tag to enclose self-contained elements such as Blog posts, widgets, or new’s posts.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                    <li><a href="./html/nosotros.html">Nosotros</a></li>
+                    <li><a href="./html/proyectos.html">Proyectos</a></li>
+                    <li><a href="./html/servicios.html">Servicios</a></li>
+                    <li><a href="./html/contacto.html">Contacto</a></li>
+                </ul>
+            </div>
+            <div class="footer__copy">
+                <p>© CONEPAR A.C. 2022 | Todos los derechos reservados</p>
+            </div>
+        </footer>
```
Comment (cuauh-cabrera) [inline index.html:83]: [CHANGES REQUIRED] Well done using a footer, but keep in mind that the <main> tag does not have to enclose the <footer> tag. The footer should be after your <main>.

**Review by cuauh-cabrera — COMMENTED** [review]
Changes required :recycle: 

Highlights: :raised_hands:

- [ ] Well done using media queries to make your project responsive :heavy_check_mark:
- [ ] Your CSS stylesheet is clean and well-structured.  :heavy_check_mark:

To improve: :pencil:
- [ ] It’s better to keep your “footer” tag outside your “main”. The “main’ tag is aimed to enclose all the content you want to show to your audience and also, is the section where the Web search engines will look for information and feed their search results. 💡 

- [ ] It’s not necessary to enclose your hero section inside an “article” tag. The “article” tag is aimed to enclose self-contained content such as blog posts, news, or widgets. 💡

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
Your comments are clear and easy to understand 💯 ✔️

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @cuauh-cabrera, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Good use of inline comments ✅
- Requested changes contain an explanation why ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**